.gitignore

```diff
@@ -48,3 +48,25 @@ passphrase
 .coverage.*
 coverage.xml
 htmlcov/
+
+\.vscode/settings\.json
```

Comment:
just ignore `.vscode` instead of two different files

Reply:
;-)